service/src/test/java/com/myhome/controllers/CommunityControllerTest.java

```diff
@@ -126,6 +120,9 @@
   @Mock
   private PaymentService paymentService;
 
+  @InjectMocks
+  private AmenityController amenityController;
```

Comment:
Since `CommunityController` do not have a connection to amenity more we should not include the amenity testing logic here :) 
That should a part of `AmenityControllerTest`